apps/http-api-app/distribution/run-trusted-node.sh

```
@@ -0,0 +1,101 @@
+    IFS=',' read -ra TYPES <<< "$TRANSPORT_TYPES"
+    INDEX=0
+    for TYPE in "${TYPES[@]}"; do
+        TYPE=$(echo "$TYPE" | xargs) # trim whitespace
+        JAVA_OPTS="$JAVA_OPTS -Dapplication.network.supportedTransportTypes.$INDEX=$TYPE"
+        INDEX=$((INDEX + 1))
+    done
+fi
+
+# Determine data directory based on OS
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    DATA_DIR="~/Library/Application Support/$APP_NAME"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)